src/spn/algorithms/layerwise/README.rst

```diff
@@ -0,0 +1,193 @@
+============================================================
+torchspn - A PyTorch Layerwise Sum-Product Network Framework
+============================================================
+
```

Comment:
Is there a reason to package this separately than spflow, while it's a good idea to disentangle the tensorflow and pytorch backends, ideally the glue sticking them together should be the spflow structure.

Reply:
Woops, I initially wanted to publish it as separate pypi package until it came to my mind that we could simply include it into SPFlow. I already had the readme written out so this was just an artifact. I've pushed an update to the readme.

Reply:
Great! Thanks for the clarification!